commands/bmc.vegman

```diff
@@ -106,7 +106,7 @@ function cmd_datetime {
 # @restrict cmd_datetime_method admin
 # @doc cmd_datetime_method
 # Change time synchronization method
-#   {manual|ntp} - The synchronization method to use
+#   {HostTime|NTP} - The synchronization method to use
```

Comment:
Please avoid CamelCase in CLI commands and their arguments.
Also, let's make it `host|ntp`.

Reply:
done